net-mgmt/collectd/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=            collectd
+PLUGIN_VERSION=         0.1
+PLUGIN_COMMENT=         collectd agent
```

Comment:
It should be snappy, but a bit more elaborate (one very short sentence about what it can do)